mmv1/templates/cai2hcl/resource_converter.go.erb

```diff
@@ -0,0 +1,138 @@
+<%=lines(autogen_notice(:go, pwd))-%>
+
+package <%= product_ns.downcase -%>
+
+<%- # "goimports" is used to remove unreferenced imports. -%>
+import (
+    "fmt"
+    "log"
+    "reflect"
+    "time"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/customdiff"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/structure"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/logging"
+    "<%= import_path() -%>/tpgresource"
+    transport_tpg "<%= import_path() -%>/transport"
+    "<%= import_path() -%>/verify"
+   "github.com/GoogleCloudPlatform/terraform-google-conversion/v2/caiasset"
+   "github.com/GoogleCloudPlatform/terraform-google-conversion/v2/cai2hcl/cai2hcl"
+   cai2hclHelper "github.com/GoogleCloudPlatform/terraform-google-conversion/v2/cai2hcl/helper"
+   cai2hclConfig "github.com/GoogleCloudPlatform/terraform-google-conversion/v2/cai2hcl/config"
+   tfschema "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+   tpg "github.com/GoogleCloudPlatform/terraform-google-conversion/v2/tfplan2cai/converters/google/resources"
```

Comment:
This should be able to directly reference the beta provider instead of referencing the `resources` directory

Reply:
Removed this import (schema is now passed to constructor)